### ISSUE TITLE
Approve registrant

### ISSUE THREAD

**Issue by celli33** [MacsiDigital/laravel-zoom#32]
When sending a: 
```
$registrant = $meetingZoom->registrants()->create([
      'email' => $user->email,
      'first_name' => $user->name,
      'last_name' => $lastName ? $lastName : "Registrant"
]);
$registrant->approve(); 
 ``` 
 I always recieve: 
` Exception: 403 status code in file /var/www/html/project/vendor/macsidigital/laravel-zoom/src/Registrant.php on line 169 `

approve() is the only method where I'm having troubles to make it work right.

**Comment by Umer5** [comment]
I’m also dealing with the same problem. :(

**Comment by Umer5** [comment]
helo please share thi solution if you fined out
thanks

**Comment by celli33** [comment]
I haven't find any solution yet, but I'm making a call  to Zoom API  using this functions while a fix is not available: 

```
use Firebase\JWT\JWT;
use Illuminate\Support\Facades\Http;

 /**
     * update status of registrants to approve
     */
    public function updateRegistrant($meetingId, $email)
    {
        Http::withToken($this->generateJWT())
            ->put("https://api.zoom.us/v2/meetings/{$meetingId}/registrants/status", [
            'action' => 'approve',
            'registrants' => [
                [ "email" => $email]
            ]
        ]);
    }

 /**
     * generete temporal token for zoom api
     */
    public function generateJWT()
    {
        $token = [
            'iss' => config('zoom.api_key'),
            // The benefit of JWT is expiry tokens, we'll set this one to expire in 1 minute
            'exp' => time() + 60,
        ];
        return JWT::encode($token, config('zoom.api_secret'));
    }
```

**Comment by Umer5** [comment]
can you sow the complete code how generate access token

**Comment by colinhall17** [comment]
Hi all

Apologies in the delay in coming back to you, when I created this package it was a quick build for a project I needed it for and I made it open source in case it would help someone. 

With the increased use of Zoom around the world its gaining some traction, so I've built 3 new versions for Laravel 5.5 upwards, make sure you select the correct version for your version of Laravel.

Open a new issue with any problems.
